# Problem 13

http://www.pythonchallenge.com/pc/return/disproportional.html

In [1]:
import xmlrpclib

Connect to the remote XML-RPC server through the Python APIs:

In [2]:
server = xmlrpclib.ServerProxy('http://www.pythonchallenge.com/pc/phonebook.php')
server.system.listMethods()

['phone',
 'system.listMethods',
 'system.methodHelp',
 'system.methodSignature',
 'system.multicall',
 'system.getCapabilities']

Find information about the 'phone' method. Input is a string, output is a string

In [3]:
print server.system.methodHelp('phone')
print server.system.methodSignature('phone')

Returns the phone of a person
[['string', 'string']]


In [4]:
server.phone('Bert')

'555-ITALY'

Apparently 555 stands for 'fake number', so the solution is 'italy'.